streamlitDemo/start_webapp.py

```diff
@@ -3,12 +3,12 @@
 
 headerString = '''
 ##################################################
-#########         BrillAI Web App        #########
+#########         Brilla AI Web App        #########
```

Comment:
code here doesn't import `APP_NAME` from `utility.py` as that will result in an error. `start_webapp.py` is supposed to build all the dependencies ad start the app so importing from `utility.py` won't work as libraries used in that file haven't yet been installed